### PR TITLE
fleet: initial document for fleet-with-grpc

### PR DESCRIPTION
**Pull request by dongsupark**
To avoid confusion about setting up gRPC development environment for fleet, I've come up with a short instruction on how to set it up.

/cc @hectorj2f

### CODE REVIEW

File: fleet/fleet-with-grpc.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Fleet with gRPC
+
+To build fleet with gRPC enabled, first of all, you need to install several
+protobuf compilers.
+
+## Install protobuf packages
+
+Install a native protobuf compiler v3.0.
```
Comment (hectorj2f) [inline fleet/fleet-with-grpc.md]: @dongsupark I'd say that we used v3.0 but other versions should work too. It seems it is v3.0 specific based on what you mention.

Comment (hectorj2f) [inline fleet/fleet-with-grpc.md]: I'd add a link to protobuf documentation how to build it.

File: fleet/fleet-with-grpc.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+$ git checkout -b c3995ae4 c3995ae4
+$ make install
+```
+
+## Convert *.proto to *.pb.go
+
+Now you need to clone git repo of fleet, to check out a specific branch to enable gRPC.
+
+```
+$ go get -u github.com/endocode/fleet
```
Comment (hectorj2f) [inline fleet/fleet-with-grpc.md]: Shouldn't it be `coreos/fleet` ?

File: fleet/fleet-with-grpc.md (hunk 1 above)
Comment (dongsupark) [inline fleet/fleet-with-grpc.md]: @hectorj2f okay, then I'd change it to "Install a native protobuf compiler. Any version should work, but v3.0 is recommended."
I'll also add a link to protobuf documentation.

File: fleet/fleet-with-grpc.md (hunk 2 above)
Comment (dongsupark) [inline fleet/fleet-with-grpc.md]: At the moment, there's no grpc branch on ```github.com/coreos/fleet```. The experimental grpc branches are only on ```github.com/endocode/fleet``` or ```github.com/giantswarm/fleet```.
Yeah, I agree, it is confusing. Once we manage to get it merged to master, we won't need to care about checking out those branches, etc.

**Comment by hectorj2f** [comment]
LGTM

**Comment by dongsupark** [comment]
@hectorj2f Thanks. I just updated it to add a description about the protobuf version, as well as a link to the protobuf documentation.

**Comment by jonboulle** [comment]
Can't we just codify this in the genproto script or similar? I'd much rather have it in a reproducible runnable script than a series of copy/paste steps..

**Comment by zbwright** [comment]
Thanks for your work. fleet is no longer actively developed or maintained by CoreOS. Closing this PR.

**Pull request closed without merging**